packages/yoga/src/heading/web/Title.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Text from '../../Text';
+import Box from '../../Box';
+
+const Title = ({ children }) => (
+  <Box w="100%" textAlign="center">
+    <Text.Medium size="small">{children}</Text.Medium>
```

Comment:
I think we should use the h1 tag by default, it is common to use this text as the h1 of the page.
We could pass the other props to h1, that way it would be possible to change its `aria-level`.